virttest/qemu_vm.py

```diff
@@ -2262,6 +2263,23 @@ def sort_key(dev):
 
         return devices, spice_options
 
+    def _br_tap_cleanup(self, nic):
+        def _del_tap(nic):
+            br_mgr, br_name = utils_net.find_current_bridge(nic.ifname)
+            if br_name == nic.netdst:
+                br_mgr.del_port(nic.netdst, nic.ifname)
+
+        if nic.ifname not in utils_net.get_net_if():
+            _del_tap(nic)
+        else:
+            # NIC with multiple queues request longer deletion time.
+            deletion_time = math.ceil(self.params.get('queues', 1) / 2)
```

Comment:
Suggest `deletion_time = max(5, math.ceil(self.params.get('queues', 1) / 2))`

Reply:
Or simply `deletion_time = max(5, self.params.get('queues', 1) // 2 + 1)`

Reply:
Updated. Thanks for review.